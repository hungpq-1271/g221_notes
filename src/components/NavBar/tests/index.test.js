import React from 'react'
import { render } from '@testing-library/react'
import 'jest-styled-components'

import NavBar from '../index'

describe('<NavBar />', () => {
  it('should render and match the snapshot', () => {
    const { container } = render(<NavBar />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
