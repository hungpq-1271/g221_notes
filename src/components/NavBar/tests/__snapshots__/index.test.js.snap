// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> should render and match the snapshot 1`] = `
.c0 {
  backgroud-color: black;
  color: white;
  font-family: Roboto;
}

.c1 {
  margin-left: auto;
}

<div
  class="NavBar"
>
  <header
    class="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionFixed c0 MuiAppBar-colorPrimary mui-fixed"
  >
    <div
      class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h4
        class="MuiTypography-root MuiTypography-h4"
      >
        G221 Notes&Tasks
      </h4>
      <button
        class="MuiButtonBase-root MuiButton-root c1 MuiButton-text"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label"
        >
          Login
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </header>
</div>
`;
